### ISSUE TITLE
Add possibility to run AsyncApiFilter implementation in a build time. Similar to the @OpenApiFilter( OpenApiFilter.RunStage.BOTH ) in Open API extension. 

### ISSUE THREAD

**Issue by td83** [quarkiverse/quarkus-asyncapi#199]
It would be very helpful to add the possibility of running the filter at compile time, as is possible in the Open API extension. This is because it is often necessary to add new schemas to a document and for this you usually need access to the Jandex index.

**Comment by ChMThiel** [comment]
Good idea! We use this openApi-filter features in our project as well.
Unfortunately i'm currently busy, don't know when i will have time for this.
